web/js/modules/location-search/actions.js

```diff
@@ -98,7 +102,7 @@ export function setPlaceMarker(coordinates, reverseGeocodeResults, isInputSearch
 export function clearCoordinates() {
```

Comment:
This should no longer be needed unless we expect to add functionality to remove all markers at some point.